### DISCUSSION TITLE
Jeedom Luna passé au crible - Haade.fr

### DISCUSSION THREAD

**Discussion by giscus** [haade-administrator/haade.fr#12] [Comments]
# Jeedom Luna passé au crible - Haade.fr

Je vais examiner Jeedom Luna dans le détail, en rechercher les origines, les causes ou encore les avantages, les inconvénients ou les problèmes, bref que se cache-t-il sous la coque de la jeedom Luna

https://haade.fr/fr/blog/jeedom-luna-passe-au-crible

**Comment by cybermil62** [comment]
Bonjour, 
Merci pour cet article très complet et très intéressant. Je suis moi même utilisateur de Jeedom de longue date (Jeedom Mini+). J’avais laissé la Box dans un carton pendant plusieurs années et ai voulu la remettre en route. Malheureusement, j’ai compris qu’il était très difficile de continuer à s’en servir sans mise à jour. 
Je suis tenté de migrer vers le modèle Luna, mais quid de la Box Mini+, est-il possible de la recycler ? (Émulateur ou autre?). 
Le port nanoSIM de la Luna permet t-il d’avoir une connexion de secours en cas de coupure de courant afin de maintenir la connectivité vers l’extérieur ? 
Hervé

**Reply by haade-administrator** [reply]
la jeedom mini+ manque effectivement de puissance, mais comme tous raspberry, elle est recyclable, tu peux l'utiliser pour installer un système vocal par exemple comme rhasspy, il y a tellement de chose à faire qu'elle trouvera bien une utilité tant que le système installé dessus n'absorbe pas trop de puissance.
Pour la luna le port nanoSIM permet d'accueillir une carte 4G, je crois que pour l'instant l'intégration n'est pas prise en compte sur jeedom mais ça va venir prochainement.
